src/workflows/index.js

```diff
@@ -1,10 +1,10 @@
-import CodeSaturn   from './code-saturn';
+// import CodeSaturn   from './code-saturn';
 import PyFr         from './pyfr/pyfr-simput';
 import PyFrExec     from './pyfr/pyfr-exec';
 import Visualizer   from './visualizer';
 
 const Workflows = {
-  CodeSaturn,
+  // CodeSaturn,
   PyFr,
   PyFrExec,
   Visualizer,
```

Comment:
Yes, the beauty of using a VCS is nothing is really gone, we can always revert this commit in the future.